news/2023/2023-07-21-skinning-contest-icons-of-an-era-voting.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Skinning Contest: Icons of an Era - Voting Open"
+date: 2023-07-21 21:00:00 +0000
+---
+
+Summer is the best part of the year to indulge in some nostalgia, so why not join us for a trip down memory lane and vote for the best skins that showcase the past 15 years of osu!?
+
+![](/wiki/shared/news/2023-07-21-skinning-contest-icons-of-an-era-voting/banner.jpg)
+
+Tablet pencils down everybody! Submissions have been closed, the skins have been screened, and it is now time for the community to vote on their favourite skins from this iteration of the skinning contest. If you are somehow unaware of this contest and what the theme was, feel free to check out the [announcement news post](https://osu.ppy.sh/home/news/2023-05-10-skinning-contest-icons-of-an-era).
+
+**16 talented artists** have put their blood, sweat and tears (mostly sweat) into their creations, come up with some great skin ideas and are now eagerly waiting to receive one of your votes! You can find all **16 entries** on [the completed skins compendium](https://compendium.skinship.xyz/contests/4), we suggest that you download and play the game with all the entries before casting your votes. If you are not interested in downloading every single entry, feel free to check out skin previews by the [skinship](https://skinship.xyz/) team in the embedded playlist below:
```

Comment:
> You can find all **16 entries** on [the completed skins compendium]

Kinda makes it sound like you can only find them there, which is not the case.. Its merely an easy way to take a look at them and see all the screenshots+videos without needing to parse every forum post

Reply:
we apparently also did it like this last time, guess its fine

Reply:
rock said its fine